src/sql_src.sql

```diff
@@ -14,3 +32,205 @@ BEGIN
   END IF;
 END;
 $$ LANGUAGE plpgsql;
+
+------------------------------------------------------------
+-- Functions
+------------------------------------------------------------
```

Comment:
It feels pretty bad to simply copy paste those functions but I don't see a better way.